backend/app.js

```diff
@@ -54,8 +54,37 @@ app.post('/changeUsername', (req, res) => {
 })
 
 // TODO T32: log in 
-app.post('/login', (req, res) => {
-    res.status(200).send("Login endpoint under construction.");
+app.post('/login', async (req, res) => {
+    const { email, password } = req.body;
+
```

Comment:
Add an if statement to check if the email and/or password is undefined. Send back a http 400 status code for bad request.

Reply:
check out this validator https://express-validator.github.io/docs/